dot/parachain/candidate-validation/candidate_validation.go

```diff
@@ -32,6 +31,7 @@ type CandidateValidation struct {
 	SubsystemToOverseer chan<- any
 	OverseerToSubsystem <-chan any
 	ValidationHost      parachainruntime.ValidationHost
+	RuntimeInstance     parachainruntime.RuntimeInstance
```

Comment:
That may be a better way to go, I'll take a look at this as I'm implementing the PVF host.

Reply:
> That may be a better way to go, I'll take a look at this as I'm implementing the PVF host.

It is easy to get such comments lost. Using runtime instance directly is a mistake here. There is no reason to merge that and postpone the fix to some PR that might not be directly relevant. I would suggest we do it as part of this PR.
If this was a bigger work, we could have thought of doing it separately, but from what I understand it would not take long.

Reply:
Good points, I've updated this in this PR.